project4/name_server/nameserver.h

```diff
@@ -46,20 +45,19 @@ class Nameserver {
    */
   void Exit();
 
-  /**
-   * @brief Handles a NameserverMessage
-   * @param msg the message
-   */
-  void HandleRequest(const consistent_hash_msgs::NameServerMessage &msg, const message_passing::Endpoint source);
-
-  /// The server object
-  std::unique_ptr<message_passing::Server> server_;
+  virtual /**
```

Comment:
(minor) This is some really weird formatting.

Reply:
This was unintentional. Fixed.